engine-standalone-storage/src/sync/types.rs

```diff
@@ -476,6 +479,7 @@ impl<'a> TryFrom<BorshableTransactionMessage<'a>> for TransactionMessage {
 #[derive(BorshDeserialize, BorshSerialize, Clone)]
 enum BorshableTransactionKind<'a> {
     Submit(Cow<'a, Vec<u8>>),
+    SubmitWithArgs(Cow<'a, parameters::SubmitArgs>),
```

Comment:
> I think it would be nice to expand the standalone helpers used in the tests to also handle this new function. See [standalone validation logic](https://github.com/aurora-is-near/aurora-engine/blob/develop/engine-tests/src/test_utils/mod.rs#L225-L230) in AuroraRunner which uses the [submit_raw function](https://github.com/aurora-is-near/aurora-engine/blob/develop/engine-tests/src/test_utils/standalone/mod.rs#L186). https://github.com/aurora-is-near/aurora-engine/pull/690 also did a similar change.

But it seems already [done](https://github.com/aurora-is-near/aurora-engine/pull/696/files#diff-f65b2de46a5b104b420fca770804b31790e281f48b886d7fe687c8fd9e43fe43R213) No?

Reply:
> But it seems already done

Yes, my apologies. I somehow missed it!